aten/src/ATen/core/TensorBase.h

```diff
@@ -203,10 +203,7 @@ class TORCH_API TensorBase {
     impl_.reset();
   }
 
-  TensorBase& operator=(const TensorBase& x) & {
-    impl_ = x.impl_;
-    return *this;
-  };
+  TensorBase& operator=(const TensorBase& x) & = default;
```

Comment:
MSVC though a few over this last time I tried it FYI

Reply:
```suggestion
  TensorBase& operator=(const TensorBase& x) noexcept & = default;
``` 
may work but i haven't tested it.

Reply:
MSVC screamed last time I tried this, but I refactored the other assignment operator since then so maybe it will work.